src/server.py

```diff
@@ -730,7 +729,17 @@ def _resume_last_bg_player(self, uuid_):
                 "background sounds, but this isn't", uuid)
 
         if len(self._background_players) > 0:
-            self._background_players[-1].play()
+            last_player = self._background_players[-1]
+            self.logger.info("Resuming player.",
+                             sound_event_id=last_player.sound_event_id,
+                             uuid=last_player.uuid)
+            if self.refcount(last_player.uuid) == 0:
+                self.logger.info("Cannot resume this player because it is "
+                                 "unreferred.",
```

Comment:
I would say something like "its owning app has disappeared from the bus", here it's not clear on a casual reading if it was unreferenced from inside the server or externally.